.github/workflows/simplesecurity.yml

```diff
@@ -0,0 +1,26 @@
+name: simplesecurity
```

Comment:
Considering that this is in the `/.github` folder, what kind of behaviour would you expect?

Reply:
My issue is coupling too closely to a specific git provider. What happens if in a few years github does a big bad and falls out of favour. Being more portable is in my interest rather than learning a specific yaml schema to tie one to a specific service